client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -44,9 +54,7 @@ export class PaidPlanThankYouCard extends Component {
 										{ translate( 'So long spam, hello backups!' ) }
 									</h1>
 									<p>
-										{ translate(
-											'We’ve finished setting up spam filtering and backups for you.'
-										) }
+										{ translate( 'We’ve finished setting up spam filtering and backups for you.' ) }
```

Comment:
Sorry, not new but prettier touched this. Inconsistent punctuatinon:

```suggestion
										{ translate( "We've finished setting up spam filtering and backups for you." ) }
```